libnavigator/src/test/java/com/mapbox/navigation/navigator/ActiveGuidanceOptionsMapperParameterizedTest.kt

```diff
@@ -0,0 +1,366 @@
+                    Waypoint(Point.fromLngLat(3.456, 7.891), true),
+                    Waypoint(Point.fromLngLat(4.567, 8.912), true),
+                    Waypoint(Point.fromLngLat(5.678, 9.123), false)
+                )
+            ),
+            arrayOf(
+                DirectionsCriteria.PROFILE_CYCLING,
+                DirectionsCriteria.GEOMETRY_POLYLINE,
+                listOf(0, 4),
+                coordinatesList,
+                ActiveGuidanceMode.KCYCLING,
+                ActiveGuidanceGeometryEncoding.KPOLYLINE5,
+                listOf(
+                    Waypoint(Point.fromLngLat(1.234, 5.678), false),
+                    Waypoint(Point.fromLngLat(2.345, 6.789), true),
+                    Waypoint(Point.fromLngLat(3.456, 7.891), true),
+                    Waypoint(Point.fromLngLat(4.567, 8.912), true),
+                    Waypoint(Point.fromLngLat(5.678, 9.123), false)
+                )
+            ),
+            arrayOf(
+                DirectionsCriteria.PROFILE_CYCLING,
+                DirectionsCriteria.GEOMETRY_POLYLINE6,
+                listOf(0, 1, 4),
+                coordinatesList,
+                ActiveGuidanceMode.KCYCLING,
+                ActiveGuidanceGeometryEncoding.KPOLYLINE6,
+                listOf(
+                    Waypoint(Point.fromLngLat(1.234, 5.678), false),
+                    Waypoint(Point.fromLngLat(2.345, 6.789), false),
+                    Waypoint(Point.fromLngLat(3.456, 7.891), true),
+                    Waypoint(Point.fromLngLat(4.567, 8.912), true),
+                    Waypoint(Point.fromLngLat(5.678, 9.123), false)
+                )
+            ),
+            arrayOf(
+                DirectionsCriteria.PROFILE_CYCLING,
+                "geojson",
+                listOf(0, 2, 4),
+                coordinatesList,
+                ActiveGuidanceMode.KCYCLING,
+                ActiveGuidanceGeometryEncoding.KGEO_JSON,
+                listOf(
+                    Waypoint(Point.fromLngLat(1.234, 5.678), false),
+                    Waypoint(Point.fromLngLat(2.345, 6.789), true),
+                    Waypoint(Point.fromLngLat(3.456, 7.891), false),
+                    Waypoint(Point.fromLngLat(4.567, 8.912), true),
+                    Waypoint(Point.fromLngLat(5.678, 9.123), false)
+                )
+            ),
+            arrayOf(
+                DirectionsCriteria.PROFILE_CYCLING,
+                "unrecognized",
+                listOf(0, 3, 4),
+                coordinatesList,
+                ActiveGuidanceMode.KCYCLING,
+                ActiveGuidanceGeometryEncoding.KPOLYLINE6,
+                listOf(
+                    Waypoint(Point.fromLngLat(1.234, 5.678), false),
+                    Waypoint(Point.fromLngLat(2.345, 6.789), true),
+                    Waypoint(Point.fromLngLat(3.456, 7.891), true),
+                    Waypoint(Point.fromLngLat(4.567, 8.912), false),
+                    Waypoint(Point.fromLngLat(5.678, 9.123), false)
```

Comment:
Could we extract some "magic numbers" into constants?

Reply:
Same as above